apis/userapi/userapi.yaml

```diff
@@ -1730,39 +1789,92 @@ paths:
             'INTERNAL SERVER ERROR. We track these errors automatically and try to set it right at the earliest. Try refreshing the page. If the problem persists contact us at info@sunbird.org.'

```

Comment:
INTERNAL SERVER ERROR, we are tracking the error and cater to it on priority. Try to refresh the page and if the problem persists contact us at info@sunbird.org